lectures/Elsäkerhet.tex

```diff
@@ -0,0 +1,441 @@
+\begin{center}
+	\begin{minipage}{0.19\columnwidth}
+		\Huge{\fontencoding{U}\fontfamily{futs}\selectfont\char 66\relax}
+	\end{minipage}
+	\begin{minipage}{0.7\columnwidth}
+		Om du är osäker på det elsäkerhetsmässiga utförandet på en
+		apparat, till exempel en gammal sändare, använd då en skiljetransformator
+		(fulltransformator) -- helst av klass II (extraisolerad).
+	\end{minipage}
+\end{center}
+\end{frame}
+
+\begin{frame}{Reparation utan spänning}
+Vid reparation ska utrustningen vara spänningslös.
+Före arbetet ska du:
+
+\begin{enumerate}
+	\item stänga av utrustningens nätströmbrytare
+	\item dra ur stickproppen ur vägguttaget (dubbel säkerhet).
+\end{enumerate}
+\end{frame}
+
+\begin{frame}{Felsökning med spänning sid 1/2}
+Om trimning eller felsökning måste ske under spänning ska följande iakttas:
+\begin{itemize}
+	\item Arbeta inte med anläggningen när du är trött eller omotiverad.
+	Då är du minst vaksam mot olyckor.
+	\item Se till att du inte får ström genom kroppen, arbeta helst bara med en
+	hand och håll den andra borta från den utrustning som du arbetar med.
+	Stoppa gärna den fria handen i fickan!
+\end{itemize}
+\end{frame}
+
+\begin{frame}{Felsökning med spänning sid 2/2}
+\begin{itemize}
+	\item Ha inga hörtelefoner på huvudet.
+	Använd högtalare om du trimmar med hörseln.
+	\item Helst bör någon finnas i närheten när du arbetar i apparater under
+	spänning.
+	Visa var nätströmbrytaren sitter.
+	Se gärna till att han/hon kan elolycksfallshjälp.
+\end{itemize}
+\end{frame}
+
+\begin{frame}{Ackumulatorbatterier sid 1/2}
+Vid arbete med ackumulatorbatterier:
+
+\begin{itemize}
+	\item Trots att spänningen är låg kan ackumulatorbatterier lämna
+	mycket höga strömmar vid kortslutning.
+	Tag därför av fingerringar, armbandsur med mera.
+	
+	Använd isolerade verktyg vid arbete med batteripolskor.
+	\item Akta dig för elektrolyten i ackumulatorbatterierna -- den är
+	starkt frätande.
+\end{itemize}
+\end{frame}
+
+\begin{frame}{Ackumulatorbatterier sid 2/2}
+\begin{itemize}
+	\item Varning för explosionsrisk av knallgas och syrastänk i ögonen.
+	\item Moderna litium-polymer (LiPo) batterier är oerhört energirika.
+	Dessa kan börja brinna med hög temperatur, och bör behandlas varsamt samt
+	läggas i därför lämpliga skyddspåsar.
+	Olika varianter är olika känsliga, så det är rekommenderat att läsa på om
+	de alternativ som finns och hur bäst hantera dem.
+	Akta för överladdning!
+\end{itemize}
+\end{frame}
+
+\begin{frame}{Antenner sid 1/5}
+Placera helst antennerna utom räckhåll för obehöriga.
+På sändarantenner kan det nämligen uppstå höga HF-spänningar redan vid
+låg sändareffekt.
+HF bränns vid beröring och en reflexrörelse gör det lätt att tappa balansen och
+falla.
+Sätt gärna upp skyltar på eller invid antennerna, med varning för högfrekvent
+spänning samt uppgift om ägarens namn, adress och telefonnummer.
+\end{frame}
+
+\begin{frame}{Antenner sid 2/5}
+En obalanserad antenn kan resultera i stor spänning även på ansluten
```

Comment:
Det ska in ett komma här för att få rätt "flow" på meningen. Läser man hela meningen så flyter det på bättre.
```suggestion
En obalanserad antenn kan resultera i stor spänning, även på ansluten
```

Reply:
Har formulerat om texten
En obalanserad antenn kan resultera i stor spänning även på ansluten
antennkabel.
Att röra antennkabeln kan därför innebära samma risker som att ta i själva
antennen.